### PR TITLE
Test infrastructure improvements for several tests

### PR DESCRIPTION
**Pull request by byron-marohn**
Tests improved: hash_test, hash_perf_test, pktdev_test, pkt_test, acl_test

- Improves #234
- Replaced cne_printf() with tst_*()
- Changed some output to improve readability

Signed-off-by: Byron Marohn <byron.marohn@intel.com>
Signed-off-by: Stephen Weeks <stephen.weeks@intel.com>
Signed-off-by: Matthew Leon <matthew.leon@intel.com>
Signed-off-by: Edmund Leemhuis <edmund.leemhuis@intel.com>

### CODE REVIEW

File: test/testcne/acl_test.c [hunk 1]
```diff
@@ -370,12 +373,13 @@ test_classify_buid(struct cne_acl_ctx *acx, const struct cne_acl_ipv4vlan_rule *
 static int
 test_classify(void)
 {
+    tst_info("%s(%s)", __func__, "Test scalar and SSE ACL lookup");
```
Comment (KeithWiles) [inline test/testcne/acl_test.c:376]: Please move the tst_info() line to after the function declaration or above line 380.

File: test/testcne/acl_test.c [hunk 2]
```diff
@@ -420,18 +424,19 @@ test_classify(void)
 static int
 test_classify_avx2(void)
 {
+    tst_info("%s(%s)", __func__, "Test avx2 ACL lookup");
```
Comment (KeithWiles) [inline test/testcne/acl_test.c:427]: Same here move to before line 431

File: test/testcne/acl_test.c [hunk 3]
```diff
@@ -477,18 +482,19 @@ test_classify_avx2(void)
 static int
 test_classify_avx512x16(void)
 {
+    tst_info("%s(%s)", __func__, "Test avx512x16 ACL lookup");
```
Comment (KeithWiles) [inline test/testcne/acl_test.c:485]: Same here

File: test/testcne/acl_test.c [hunk 4]
```diff
@@ -534,18 +540,19 @@ test_classify_avx512x16(void)
 static int
 test_classify_avx512x32(void)
 {
+    tst_info("%s(%s)", __func__, "Test avx512x16 ACL lookup");
```
Comment (KeithWiles) [inline test/testcne/acl_test.c:543]: same here

File: test/testcne/acl_test.c [hunk 5]
```diff
@@ -589,6 +596,8 @@ test_classify_avx512x32(void)
 static int
 test_build_ports_range(void)
 {
+    tst_info("%s(%s)", __func__, "Test ports ACLs");
```
Comment (KeithWiles) [inline test/testcne/acl_test.c:599]: same here

File: test/testcne/acl_test.c [hunk 6]
```diff
@@ -1047,6 +1056,7 @@ test_convert_rules(const char *desc, void (*config)(struct cne_acl_config *),
 static int
 test_convert(void)
 {
+    tst_info("%s(%s)", __func__, "Test convert rules");
```
Comment (KeithWiles) [inline test/testcne/acl_test.c:1059]: same here

File: test/testcne/acl_test.c [hunk 7]
```diff
@@ -1108,6 +1118,7 @@ test_convert(void)
 static int
 test_invalid_layout(void)
 {
+    tst_info("%s(%s)", __func__, "Test wrong layout behavior");
```
Comment (KeithWiles) [inline test/testcne/acl_test.c:1121]: same here

File: test/testcne/acl_test.c [hunk 8]
```diff
@@ -1222,6 +1234,7 @@ test_invalid_layout(void)
 static int
 test_create_find_add(void)
 {
+    tst_info("%s(%s)", __func__, "Test creating and finding ACL contexts");
```
Comment (KeithWiles) [inline test/testcne/acl_test.c:1237]: same here

File: test/testcne/acl_test.c [hunk 9]
```diff
@@ -1312,14 +1326,15 @@ test_create_find_add(void)
 static int
 test_invalid_rules(void)
 {
+    tst_info("%s(%s)", __func__, "test various invalid rules");
```
Comment (KeithWiles) [inline test/testcne/acl_test.c:1329]: same here

File: test/testcne/acl_test.c [hunk 10]
```diff
@@ -1405,6 +1420,7 @@ test_invalid_rules(void)
 static int
 test_invalid_parameters(void)
 {
+    tst_info("%s(%s)", __func__, "test functions by passing invalid params");
```
Comment (KeithWiles) [inline test/testcne/acl_test.c:1423]: same here

File: test/testcne/acl_test.c [hunk 11]
```diff
@@ -1614,6 +1630,7 @@ test_invalid_parameters(void)
 static int
 test_misc(void)
 {
+    tst_info("%s(%s)", __func__, "Various tests to improve coverage.");
```
Comment (KeithWiles) [inline test/testcne/acl_test.c:1633]: same here

**Review by KeithWiles — CHANGES_REQUESTED** [review]
A few minor changes and it is looking good.

File: test/testcne/acl_test.c [hunk 12]
```diff
@@ -337,6 +337,9 @@ test_classify_run(struct cne_acl_ctx *acx, struct ipv4_7tuple test_data[], size_
 err:
     /* swap data back to cpu order so that next time tests don't fail */
     bswap_test_data(test_data, dim, 0);
+    if (ret != 0) {
```
[the remark this answers is not in the document]
Comment (byron-marohn) [reply test/testcne/acl_test.c]: Fixed, this was the only one in this patch set.

File: test/testcne/acl_test.c (hunk 1 above)
Comment (byron-marohn) [reply test/testcne/acl_test.c]: Fixed all of these being before variable declarations in this file. I don't believe other files had this same problem.

**Comment by byron-marohn** [comment]
Fixed review suggestions & rebased

**Review by KeithWiles — APPROVED** [review]
LGTM, thank you.

**Pull request merged**